### ISSUE TITLE
[IMPROVEMENT] Precompile SPDK binary for longhorn-instance-manager

### ISSUE THREAD

**Issue by derekbit** [longhorn/longhorn#5903]
## Is your improvement request related to a feature? Please describe (👍 if you like this request)

Currently, the process of building SPDK binaries together with longhorn-instance-manager is time-consuming and inconvenient when attempting to build instances for debugging purposes. To optimize time efficiency, a possible solution is to precompile the SPDK binaries beforehand and then directly copy them into the longhorn-instance-manager.

**Comment by derekbit** [comment]
We have introduced multi-stage build which can reduce the build time if an application is already built. The request is unnecessary anymore.

cc @innobead

**Comment by innobead** [comment]
Sounds good. Besides, the most important thing is we should not have any build stuff for SPDK persistent in the final runtime image. Do we?

**Comment by derekbit** [comment]
Yes, we only copy the binaries and setup script from the build image to the release image, so there is no build stuff in the release image.

**Comment by innobead** [comment]
That's great. Then, we can close this with won't fix.

BTW, IIRC, we should have a ticket to improve not relying on the setup script directly, as it will install some unnecessary dependencies.
